iati/guidance_and_support/models.py

```diff
@@ -1,7 +1,70 @@
 from django.db import models
 
+from wagtail.core.fields import StreamField
 from wagtail.core.models import Page
+from wagtail.images.edit_handlers import ImageChooserPanel
 
+from home.models import AbstractContentPage, AbstractIndexPage, IATIStreamBlock
 
-class GuidanceAndSupportPage(Page):
-    pass
+
+class GuidanceAndSupportPage(AbstractContentPage):
+    """A base for the Guidance and Support page."""
+    parent_page_types = ['home.HomePage']
+    subpage_types = ['guidance_and_support.GuidanceGroupPage', 'guidance_and_support.KnowledgebaseIndexPage']
+
+    @property
+    def guidance_groups(self):
+        """Get all GuidanceGroupPage objects that have been published."""
+        guidance_groups = GuidanceGroupPage.objects.child_of(self).live()
+        return guidance_groups
+
+
+class GuidanceGroupPage(AbstractContentPage):
```

Comment:
Is there a way we can add some help text (or some other hint) to the `CONTENT EDITOR` section of the CMS (for this page only) to indicate that the content will appear on the page itself...?

Reply:
Done, just overrode the inherited field.